src/analytics/Properties.tsx

```diff
@@ -1189,7 +1189,11 @@ interface DappEventProperties extends DappProperties {
 
 interface DappExplorerEventsProperties {
   [DappExplorerEvents.dapp_impression]: DappEventProperties
-  [DappExplorerEvents.dapp_open]: DappEventProperties
+  [DappExplorerEvents.dapp_open]: DappEventProperties & {
+    activeFilter?: string
+    activeSearchTerm?: string
+    position?: number
```

Comment:
i played around with various typescript types for this, to see if we can get static typechecking (e.g. for certain dapp sections, these 3 props are required but for other dapp sections, they are optional). it became complex and brittle, so i've settled for unit tests to cover the cases we care about